setup.py

```diff
@@ -130,8 +130,8 @@ def spirv_compile():
         "numba_dpex/ocl/atomics/atomic_ops.spir",
         "numba_dpex/ocl/atomics/atomic_ops.bc",
     ]
-    subprocess.check_call(clang_args, stderr=subprocess.STDOUT, shell=False)
```

Comment:
Is this change still needed?

Reply:
Based on discussions, I have reopened #772. Let us merge that separate PR rather then include the unrelated change here.